wpimath/src/main/java/edu/wpi/first/math/controller/PIDController.java

```diff
@@ -141,6 +144,18 @@ public void setD(double kd) {
     m_kd = kd;
   }
 
+  /**
+   * Sets the IZone range.
+   *
+   * @param izone izone range
+   */
```

Comment:
This should have an explanation of what the i zone does and how to disable it (set to 0?)